generators/ci-cd/templates/github-ci.yml.ejs

```diff
@@ -0,0 +1,69 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+name: Application CI
+on: [push, pull_request]
+jobs:
+    applications:
+        name: <%= baseName %> test suite
+        runs-on: ${{ matrix.os }}
+        timeout-minutes: 40
+        strategy:
+            fail-fast: false
+            matrix:
+                node_version: [<%= NODE_VERSION %>]
+                os: [ubuntu-latest]
+                java_version: ['8.x', '11.x']
+        env:
+            NODE_VERSION: ${{ matrix.node_version }}
+            SPRING_OUTPUT_ANSI_ENABLED: NEVER
+            SPRING_JPA_SHOW_SQL: false
+            JHI_DISABLE_WEBPACK_LOGS: true
+            NG_CLI_ANALYTICS: false
+        steps:
+            - uses: actions/checkout@v1
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ matrix.node_version }}
+            - uses: actions/setup-java@v1
+              with:
+                  java-version: ${{ matrix.java_version }}
+            - name: Stop MySQL server
+              run: sudo /etc/init.d/mysql stop
+            - name: Install node.js packages
+              run: <%= clientPackageManager %> install
+            - name: Run backend test
+              <%_ if (buildTool === 'maven') { _%>
+              run: |
+                 chmod +x mvnw
+                 ./mvnw -ntp clean verify <% if (!skipClient) { %>-P-webpack<% } %>
+              <%_ } else if (buildTool === 'gradle') { _%>
+              run: |
+                 chmod +x gradlew
+                 ./gradlew clean test integrationTest <% if (!skipClient) { %>-x webpack<% } %>
+              <%_ } _%>
+            <%_ if (!skipClient) { _%>
+            - name: Run frontend test
+              run: <%= clientPackageManager %> run <%= frontTestCommand %>
+            <%_ } _%>
+            - name: Package application
+              <%_ if (buildTool === 'maven') { _%>
+              run: ./mvnw -ntp package -Pprod -DskipTests -DskipIT
```

Comment:
are you sure about skipIT ?

Reply:
Indeed, it should be `-DskipITs`. 
Anyway, according to https://maven.apache.org/surefire/maven-failsafe-plugin/examples/skipping-tests.html, `-DskipTests` should prevent all tests to be run and thus is sufficient

Reply:
yeah that needs correction. On further thought, I will remove it as `verify` goal comes after `package` and we anyway don't want to run tests here.